wasm/src/bindings.cpp

```diff
@@ -325,6 +325,9 @@ class WebGLSkiaRenderer : public rive::SkiaRenderer {
 
 WebGLSkiaRenderer *makeSkiaRenderer(int width, int height) {
   GrContextOptions options;
+  // We don't seem to always use high resolution canvases yet (via window.devicePixelRatio).
```

Comment:
We try to detect this, but at the high-level API.
https://github.com/rive-app/rive-wasm/blob/master/js/src/rive.ts#L1435